web-jehlomat/src/utils/typography.ts

```diff
@@ -78,3 +79,11 @@ export const FormItemDescription = styled.p`
         width: 340px;
     }
 `;
+
+export const Heading2 = styled.h2`
```

Comment:
```suggestion
export const H2 = styled.h2`
```
to follow above convention

Reply:
There is already `H2` component, and also I've decided to use name `Heading2` because the component with same name is defined in Figma.

Reply:
I am not sure having both `Heading2` and `H2` globally is a good idea. Is it possible to have `Heading2` only inside of SetNewPassword scope?